network/src/commonMain/kotlin/com/wire/kalium/network/api/v8/authenticated/ClientApiV8.kt

```diff
@@ -40,4 +42,13 @@ internal open class ClientApiV8 internal constructor(
                 setBody(apiModelMapper.toApiV8(registerClientRequest))
             }
         }
+
+    override suspend fun updateClientCapabilities(
+        updateClientCapabilitiesRequest: UpdateClientCapabilitiesRequest,
+        clientID: String
+    ): NetworkResponse<Unit> = wrapKaliumResponse {
+        httpClient.put("$PATH_CLIENTS/$clientID") {
+            setBody(updateClientCapabilitiesRequest)
+        }
```

Comment:
question: will this endpoint just append or overwrite the whole capabilities

Reply:
This will append, as the endpoint will complain with an error if we try to remove a capability from it. For now, we just add as a base the 2 default capabilities starting from API v8. `consumable-notifications` and `legalhold-implicit-consent`